bot/cogs/birthday/birthday_commands.py

```diff
@@ -22,28 +24,58 @@ async def post_birthdays(self):
 
         birthdays_by_guild_id = birthday_helper.get_todays_birthdays()
         for guild_id, birthdays in birthdays_by_guild_id.items():
-            guild = self.bot.get_guild(guild_id)
-            if guild is None:
-                # Can't find guild, skip posting these birthdays
-                continue
-            birthday_channel_id = birthday_helper.get_birthday_channel_id(guild_id)
-            if birthday_channel_id is None:
-                # No birthday channel set, skip posting these birthdays
-                continue
-            channel = self.bot.get_channel(birthday_channel_id)
-            if channel is None:
-                # Can't find channel, skip posting these birthdays
-                continue
-            # Assemble birthday message
-            message = messages.birthday_message()
-            for birthday in birthdays:
-                member = guild.get_member(birthday.user_id)
-                if member is None:
-                    # Can't find member, skip this birthday
+            try:
```

Comment:
This looks like a lot of changes but I really just wrapped this section in a try/catch and then added everything below line 53 (all the month milestones)